core/internal/filestream/filestream.go

```diff
@@ -20,6 +20,7 @@ const (
 	SummaryFileName          = "wandb-summary.json"
 	OutputFileName           = "output.log"
 	defaultHeartbeatInterval = 30 * time.Second
+	DefaultTransmitInterval  = 15 * time.Second
```

Comment:
I know that it's used in a single place, but I think that keeping it with the other constants simplifies discovery (I always first come here when I need to look at it).